scripts/apps/monitoring/controllers/MonitoringController.js

```diff
@@ -37,6 +37,7 @@ export function MonitoringController($rootScope, $location, desks) {
     function preview(item) {
         vm.previewItem = item;
         vm.state['with-preview'] = !!item;
+        vm.previewItem.state === 'ingested' ? vm.showHistoryTab = false : vm.showHistoryTab = true;
```

Comment:
it seems a bit confusing that there is assignment after `?`, can you put it like:
```
vm.showHistoryTab = vm.previewItem.state !== 'ingested';
```